docs/src/cookbooks/cookbook.md

```diff
@@ -791,6 +791,8 @@ class Salt extends Module {
     val io = IO(new Bundle {})
     val drink = Module(new Coffee)
     override def desiredName = "SodiumMonochloride"
+
+    drink.io <> DontCare
```

Comment:
why did you have to add this but never before? Is there a missing check in the documentation?

Reply:
and if you are goign to add it can we just drive it to 32.U or something less magical